CMakeLists.txt

```diff
@@ -33,8 +33,11 @@ set (LOOM_INCLUDE_FOLDERS
     ${CMAKE_SOURCE_DIR}/loom/vendor/sqlite3
     ${CMAKE_SOURCE_DIR}/loom/vendor/facebook/ios/FacebookSDK.framework/Headers
     ${CMAKE_SOURCE_DIR}/loom/vendor/parse/ios/Parse.framework/Headers
+    ${CMAKE_SOURCE_DIR}/loom/vendor/dxsdk/Include
     )
 
+include_directories(${LOOM_INCLUDE_FOLDERS})
```

Comment:
I will have to test this. I removed both lines 36 and 39 at first but it wouldn't compile. Didn't really test with only line 39 gone.